topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -482,6 +482,42 @@ It is usual that the quality of the sequences is worse for the reverse than for
 >    {: .question}
 {: .hands_on}
 
+
+> ### {% icon comment %} Algorithmic details
```

Comment:
Can you use a detail box there? The detail box is expandable

Reply:
sure